exporter/awsemfexporter/metric_translator.go

```diff
@@ -164,6 +178,32 @@ func getCWMetrics(metric *pdata.Metric, metadata CWMetricMetadata, instrumentati
 	return
 }
 
+func (mt metricTranslator) translateUnit(metric *pdata.Metric) string {
+	unit := metric.Unit()
+	if descriptor, exists := mt.metricDescriptor[metric.Name()]; exists {
+		if unit == "" || descriptor.overwrite {
+			return descriptor.unit
+		}
+	}
+	switch unit {
+	case "ms":
+		unit = "Milliseconds"
+	case "s":
+		unit = "Seconds"
+	case "us":
+		unit = "Microseconds"
+	case "By":
+		unit = "Bytes"
+	case "Bi":
+		unit = "Bits"
+	default:
+		if _, supported := EMFSupportedUnits[unit]; !supported {
+			unit = "None"
```

Comment:
qq: should we keep the original unit value or change it to `None`?

Reply:
Both look okay to me. I've changed it to keeping the original.